index_release_test.go

```diff
@@ -36,6 +36,16 @@ func Test_buildReleases(t *testing.T) {
 				{
 					Active: true,
 					Authorities: []Authority{
+						{
+							Endpoint: urlMustParse("http://app-controller:8000/"),
+							Name:     "app-controller",
+							Version:  "0.1.0",
+						},
+						{
+							Endpoint: urlMustParse("http://appcatalog-controller:8000/"),
```

Comment:
```suggestion
							Endpoint: urlMustParse("http://app-controller:8000/"),
```

The URL must be the same when the running process/POD is the same.